proposals/csharp-9.0/native-integers.md

```diff
@@ -92,29 +93,30 @@ The tables below cover the conversions between special types.
 | `nint` | `int` | ExplicitNumeric | `conv.i4` / `conv.ovf.i4` |
 | `nint` | `uint` | ExplicitNumeric | `conv.u4` / `conv.ovf.u4` |
 | `nint` | `long` | ImplicitNumeric | `conv.i8` / `conv.ovf.i8` |
-| `nint` | `ulong` | ExplicitNumeric | `conv.i8` / `conv.ovf.i8` |
+| `nint` | `ulong` | ExplicitNumeric | `conv.i8` / `conv.ovf.u8` |
 | `nint` | `char` | ExplicitNumeric | `conv.u2` / `conv.ovf.u2` |
 | `nint` | `float` | ImplicitNumeric | `conv.r4` |
 | `nint` | `double` | ImplicitNumeric | `conv.r8` |
-| `nint` | `decimal` | ImplicitNumeric | `conv.i8 decimal decimal.op_Implicit(long)` |
+| `nint` | `decimal` | ImplicitNumeric | `decimal decimal.op_Implicit(long)` |
```

Comment:
Why drop the `conv.i8` from `nint -> decimal`?

Reply:
Oops. It's indeed there in implementation. Restored both. Thanks